factory/usr/lib/systemd/system/initrd-core-system-defaults.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Set gadget default configuration
+ConditionPathExists=!/run/mnt/data/system-data/_writable_defaults/.done
+ConditionPathExists=/run/mnt/data/system-data/_writable_defaults
+ConditionKernelCommandLine=|snapd_recovery_mode=install
+ConditionKernelCommandLine=|snapd_recovery_mode=recover
+After=initrd-root-fs.service
+Before=initrd-fs.service
+RequiresMountsFor=/run/mnt/data
+
+[Service]
+Type=oneshot
+ExecStart=-/bin/mkdir -p /run/mnt/data/system-data/etc
+ExecStart=/bin/cp -aT /run/mnt/data/system-data/_writable_defaults/etc /sysroot/writable/system-data/etc
```

Comment:
How was this done before this PR?

Reply:
Somewhere hidden in `handle-writable-paths` with a TODO to move it into snap-bootstrap.

Reply:
Ah no sorry. I misread.

Reply:
Actually that was also a part of `handle-writable-paths` in function `handle_writable_defaults`.

Reply:
I see it now, thanks for the pointer